guac-clearly-defined-certifier.md

```diff
@@ -0,0 +1,37 @@
+## Documentation: GUAC Pull from ClearlyDefined
```

Comment:
Let's make a few changes here. First, you don't need to say "documentation" in the header because it's on the docs site already. Second, you can replace this line with something like the below so that it will render in the CI build:

```
---
layout: page
title: ClearlyDefined certifier
permalink: /certifier-clearlydefined/
---
```